lib/views.js

```diff
@@ -5,163 +5,58 @@
  */
 
 /*
- * Get clinic based on phone number
- */
-exports.clinic_by_phone = {
-  map: function(doc) {
-    if (doc.type === 'clinic' && doc.contact && doc.contact.phone) {
-      emit([doc.contact.phone]);
-    }
-  }
-};
-
-/*
  * Get clinic based on referral id (refid) in a tasks_referral doc.
  */
-exports.clinic_by_refid = {
+exports.contacts_by_refid = {
```

Comment:
I updated the implementation to match the view name. `refid` is and external identifier that can be assigned to contacts just like our internally generated `place_id`.